pyproject.toml

```diff
@@ -1,23 +1,23 @@
-[tool.poetry]
+[build-system]
+requires = ["poetry-core>=1.0.0", "poetry-dynamic-versioning"]
+build-backend = "poetry_dynamic_versioning.backend"
+
+[project]
 name = "linkml-runtime"
-version = "0.0.0"
 description = "Runtime environment for LinkML, the Linked open data modeling language"
 authors = [
-    "Chris Mungall <cjmungall@lbl.gov>",
-    "Harold Solbrig <solbrig@jhu.edu>",
-    "Sierra Moxon <smoxon@lbl.gov>",
-    "Bill Duncan <wdduncan@gmail.com>",
-    "Harshad Hegde <hhegde@lbl.gov>"
+  {name = "Chris Mungall", email = "cjmungall@lbl.gov"},
+  {name = "Harold Solbrig", email = "solbrig@jhu.edu"},
+  {name = "Sierra Moxon", email = "smoxon@lbl.gov"},
+  {name = "Bill Duncan", email = "wdduncan@gmail.com"},
+  {name = "Harshad Hegde", email = "hhegde@lbl.gov"},
 ]
-
-readme = "README.md"
-
+license = {text = "LICENSE"}
 homepage = "https://github.com/linkml/linkml-runtime"
 repository = "https://github.com/linkml/linkml-runtime"
 documentation = "https://github.com/linkml/linkml-runtime"
-
+readme = "README.md"
```

Comment:
```suggestion
readme = { file = "README.md", content-type = "text/markdown" }
```

Reply:
I looked [it up](https://packaging.python.org/en/latest/specifications/pyproject-toml/#readme): *If the file path ends in a case-insensitive .md suffix, then tools MUST assume the content-type is text/markdown.*
So we don't need to change.

Reply:
oh that's a nice little touch :)